projects/plugins/jetpack/extensions/shared/icons.js

```diff
@@ -403,3 +403,13 @@ export const flashIcon = (
 		<Path d="M7 2v11h3v9l7-12h-4l4-8z" fill="currentColor" />
 	</SVG>
 );
+
+export const VideoPressIcon = (
+	<SVG width="29" height="21" viewBox="0 0 29 21" fill="none" xmlns="http://www.w3.org/2000/svg">
+		<Path
+			fill-rule="evenodd"
+			clip-rule="evenodd"
```

Comment:
```suggestion
			fillRule="evenodd"
			clipRule="evenodd"
```

In React context these generate a warning when not using camelCase